src/Aggregate/AggregateRootAttributeBehaviour.php

```diff
@@ -49,9 +49,11 @@ protected function apply(object $event): void
         if (count($parts) === 2) {
             [$property, $method] = $parts;
 
-            /** @var ChildAggregate $child */
-            $child = $this->$property;
-            $child->$method($event);
+            $child = $this->getChildProperty($property);
```

Comment:
The name is confusing, it's not the property that is returned but the ChildAggregate. `getChildAggregate` would be more correct here, wouldn't it?